internal/server/resource/resource.go

```diff
@@ -50,12 +50,14 @@ type Cache struct {
 
 // Metadata represents the metadata used by the server.
 type Metadata struct {
-	Mappings        []*types.Mapping
-	OutArcInfo      map[string]map[string][]types.OutArcInfo
-	InArcInfo       map[string][]types.InArcInfo
-	SubTypeMap      map[string]string
-	MixerProject    string
-	BigQueryDataset string
+	Mappings          []*types.Mapping
+	OutArcInfo        map[string]map[string][]types.OutArcInfo
```

Comment:
use pointer:

map[string]map[string][]*types.OutArcInfo

Reply:
Nice, this is a long time issue.